ua/org.eclipse.help.base/build.properties

```diff
@@ -30,5 +31,6 @@ bin.includes = doc/,\
 jars.compile.order = .,\
                      ant_tasks/helpbase-ant.jar
 source.ant_tasks/helpbase-ant.jar = src_ant/
+output.ant_tasks/helpbase-ant.jar = bin2/
```

Comment:
```suggestion
output.ant_tasks/helpbase-ant.jar = bin_ant/
```

Reply:
I had that change first, but the `.classpath` is defined for `bin2`, so that would have to be adapted as well, along with the `.gitignore`. And it would result in those resources already existing in `bin2` from previous builds to pop up for being added to the repo (as the .gitignore changes to `bin_ant`). I wanted to avoid that by sticking to the existing output folder name `bin2`, although I would also prefer your proposal of `bin_ant`. Would you prefer that we complete change to `bin_ant` and except that developers might be confused once by resources in `bin2` popping up?

Reply:
ok